apps/api/src/domain/models/shared/common-commands/grant-user-read-access/grant-resource-read-access-to-user.command.integration.spec.ts

```diff
@@ -89,7 +89,8 @@ describe('GRANT_RESOURCE_READ_ACCESS_TO_USER', () => {
         await testRepositoryProvider.testTeardown();
     });
 
-    const eventSourcedResourceTypes = [ResourceType.song];
+    // TODO: add standalone test for these resources
```

Comment:
Be sure to circle back to this.

Reply:
[Story 1](https://www.pivotaltracker.com/story/show/186327351)
[Story 2](https://www.pivotaltracker.com/story/show/186327350)